README.md

```diff
@@ -7,21 +7,27 @@ This project is the front-end interface for the World Food Programme's [PRISM pr
 ## Functionalities
 
 The new PRISM frontend is built as a static website to minimize cross dependencies and simplify deployments as much as possible. Currently, PRISM frontend provides the ability to:
+
 - Load administrative boundaries as GeoJSON (`src/config/admin_boundaries.json`)
 - Load baseline data as JSON, and link it to administrative boundaries
 - Display WMS layers from Geoserver or Open Data Cube endpoints, with date selection capabilities
 - Display CSV tables in a left side panel
 
+To chose which country to run PRISM for, you can set the environment variable `REACT_APP_COUNTRY`. Currently, available configurations are `mongolia`, `indonesia`. The default is `mongolia`.

```

Comment:
Not really required, but if its a list
- Item 1
- Item 2

Itll be easy to add to in the future.